test/integration/remove_stache_from_cart_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+class RemoveStacheFromCartTest < ActionDispatch::IntegrationTest
+  test "visitor add remove items from cart" do
+    stache = create(:stache)
+    visit staches_path
+    click_on "Add to Cart"
+    click_on "Add to Cart"
```

Comment:
Thanks for asking Mike, we wanted to add two items to the cart to make sure the quantity would be updated as well.